### PR TITLE
re-work CPH query

### PR DESCRIPTION
**Pull request by csaltmountain**

### CODE REVIEW

File: test/integration/graphql/schema.test.js [hunk 1]
```diff
@@ -195,6 +195,13 @@ describe('schema', () => {
       findBreakingChanges(
         schema,
         buildSchema(`#graphql
+          directive @auth on OBJECT | FIELD_DEFINITION
+
+          """
+          Indicates exactly one field must be supplied and this field must not be null.
+          """
+          directive @oneOf on INPUT_OBJECT
+
```
Comment (defra-dom) [inline test/integration/graphql/schema.test.js:204]: whats this for?

Comment (csaltmountain) [reply test/integration/graphql/schema.test.js]: Not sure. Must have come from when I copied the schema. I've removed it.

**Review by rkdefra — CHANGES_REQUESTED** [review]
Could you rebase rather than merge from `dev`? It would be good to keep the commit history as clear and uncluttered as possible.

File: test/unit/integration/graphql/business.test.js [hunk 2]
```diff
@@ -451,18 +445,34 @@ describe('Query.business.land', () => {
...
+              parcelNumbers
+            }
+          }
+        }
+      `,
+      schema,
+      contextValue: fakeContext
+    })
+
+    expect(result.data.business.cphList).toEqual(transformOrganisationCPH('5565448', organisationCPHFixture))
```
[the remark this answers is not in the document]
Comment (csaltmountain) [reply test/unit/integration/graphql/business.test.js:465]: This is the pattern that's used in the sort-of integration tests we have at the moment. I've added a ticket to review these tests: https://dev.azure.com/defragovuk/DEFRA-FFC/_boards/board/t/DEFRA-FFC%20Customer%20Registry/Stories?workitem=478144

File: app/graphql/types/business/cph.gql [hunk 3]
```diff
@@ -16,6 +16,23 @@ type CPHCoordinate {
 """
 Represents a County Parish Holding (CPH) number.
 
+Data Source: Rural Payments Portal (PRR)
+"""
+type CPHSummary {
+  """
+  The CPH number.
+  """
+  number: String
```
[the remark this answers is not in the document]
Comment (csaltmountain) [reply app/graphql/types/business/cph.gql:25]: Added an example to the comment to show then format of the CPH number

File: test/unit/integration/graphql/business.test.js [hunk 4]
```diff
@@ -481,16 +491,7 @@ describe('Query.business.cph', () => {
...
-      ...transformedCPH[0],
-      parish: organisationCPHInfoFixture.parish,
-      species: organisationCPHInfoFixture.species,
-      startDate: organisationCPHInfoFixture.startDate / 1000,
-      expiryDate: organisationCPHInfoFixture.expiryDate / 1000,
-      coordinate: transformOrganisationCPHCoordinates(
-        organisationCPHInfoFixture
-      )
-    })
+    expect(result.data.business.cph).toEqual(transformCPHInfo('10/327/0023', organisationCPHFixture, organisationCPHInfoFixture))
```
[the remark this answers is not in the document]
Comment (csaltmountain) [reply test/unit/integration/graphql/business.test.js:494]: As above: https://github.com/DEFRA/fcp-data-access-layer-api/pull/106#discussion_r1853646616

**Review by rkdefra — APPROVED** [review]
LGTM!

**Pull request merged**